Sources/WalletConnectSign/Engine/Controller/PairEngine.swift

```diff
@@ -20,8 +20,8 @@ actor PairEngine {
         }
         var pairing = WCPairing(uri: uri)
         try await networkingInteractor.subscribe(topic: pairing.topic)
-        let symKey = try! SymmetricKey(hex: uri.symKey) // FIXME: Malformed QR code from external source can crash the SDK
-        try! kms.setSymmetricKey(symKey, for: pairing.topic)
+        let symKey = try SymmetricKey(hex: uri.symKey)
+        try kms.setSymmetricKey(symKey, for: pairing.topic)
```

Comment:
we still need key replacement logic, right?

Reply:
correct,
it's this one:
https://github.com/WalletConnect/WalletConnectSwiftV2/issues/360